Sources/StreamChat/Database/DTOs/ChannelDTO_Tests.swift

```diff
@@ -7,6 +7,8 @@ import XCTest
 
 class ChannelDTO_Tests: XCTestCase {
     var database: DatabaseContainer!
+
+    static let channelCreatedDate = Date.unique
```

Comment:
Can you please add a comment explaining why this is needed and what problem it solves?